backend/src/graphql/resolver/TransactionLinkResolver.test.ts

```diff
@@ -50,7 +52,75 @@ afterAll(async () => {
 })
 
 describe('TransactionLinkResolver', () => {
+  describe('createTransactionLink', () => {
```

Comment:
Test for unauthenticated first?

Reply:
This test suit seems to do that only for admin queries. I'd like to leave it that way.

Reply:
May be it seems like this, but it is not like this. We test `redeemTransactionLink`  which needs no authentication. Now you test `createTransactionLink` which needs authentication

Reply:
Documented in #2704